api.rst

```diff
@@ -1227,6 +1227,28 @@ as in ``{1,2,3,4}``. Note that the curly brackets have to be urlencoded(``{`` is
 
    In these versions we recommend using function parameters of type json to accept arrays from the client.
 
+Calling variadic functions
```

Comment:
Not sure whether that whole thing actually needs it's own block or whether it can just be a note to the array part above?

Reply:
Could be a note since it works the same way as the array case. Like: "You can also call a function with a variadic parameter in this way. Example... ".

Though the downside of a note is that it doesn't generate a hash identifier for the URL.

So I think subheading is good. You could link this subheading on the `releases/upcoming.rst`(on the entry you added).